tutorials/pyam.rst

```diff
@@ -1,12 +1,26 @@
 Tutorials related to the pyam package
 =====================================
 
-.. figure:: ../../../../_static/pyam-header.png
+.. figure:: https://raw.githubusercontent.com/IAMconsortium/pyam/main/doc/logos/pyam-header.png
    :width: 640px
 
 This page collects tutorials and presentations related to the open-source Python package
 **pyam** for scenario analysis & visualization.
 
+ENGAGE Capacity Building Workshop (February 1, 2022)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+At a capacity-building workshop on good modeling practices
+organized by the `Horizon 2020 project ENGAGE`_ on February 1, 2022,
+`Daniel Huppmann <https://www.iiasa.ac.at/staff/daniel-huppmann>`_ and
+`Philip Hackstock <https://iiasa.ac.at/staff/philip-hackstock>`_
+presented an overview and live demo of the **pyam** package.
+
+ - Presentation slides: https://pure.iiasa.ac.at/id/eprint/17783/
```

Comment:
```suggestion
 - Presentation slides (english): https://pure.iiasa.ac.at/id/eprint/17783
```

Maybe we can also use the following link for the presentation slides for "Presentation at the Strommarkttreffen (July 14, 2021)" http://pure.iiasa.ac.at/id/eprint/17319 to have it in the same layout. E.g.:

- Presentation slides (german/english): http://pure.iiasa.ac.at/id/eprint/17319